packages/components/psammead-consent-banner/src/index.jsx

```diff
@@ -48,7 +44,7 @@ const hoverFocusStyles = `
 const Wrapper = styled.div`
   ${({ service }) => getSansRegular(service)}
   background-color: ${C_CONSENT_BACKGROUND};
-  border-top: solid ${transparentBorderHeight} transparent;
+  border-top: solid 0.0625rem transparent;
```

Comment:
I think this change has not been reflected in the padding calculations below, do we need seperate constants for the transparent link border (https://github.com/bbc/psammead/pull/4299/files#diff-972232a20082f3644690a9b606fed80b1787b4ad8fa8140415d5633b812636ceL81) and transparent wrapper component border here?

Reply:
Thanks @andrewscfc, good spot, I've updated the `calc`s. The transparent link border is 2px upon hover/focus (0.125rem), and only used once, happy to make a constant for this but just wondering if it's necessary if only used once. I've updated the transparent wrapper border to use the `transparentBorderHeight` constant since it's also 0.0625rem (1px), happy to change this too if you feel that another constant would be required.

Reply:
> The transparent link border is 2px upon hover/focus (0.125rem), and only used once, happy to make a constant for this but just wondering if it's necessary if only used once

If it's only used once I'd probably say it's not necessary

Changes all look good thanks 🙂